README.md

```
@@ -1,143 +1,46 @@
+Noyau is an ADE "Agentic Development Environment". It enables control of the agents on your machine with a best-in-class [Electron-based desktop app](https://noyau.hezaerd.com).
 
-Noyau v0.1 est un Environment desktop local : un Tableau Kanban par Project, des Threads Cursor
-(`Project → Thread → Turn`) et un lien optionnel `TicketThread`. Voir
-[`docs/ARCHITECTURE.md`](docs/ARCHITECTURE.md) et [l'ADR-0011](docs/adr/0011-noyau-local-first-v0.1.md).
+> [!IMPORTANT]
+> Noyau is currently in ultra-early development. The API and desktop app are subject to change.
+> 
+> Please report any issues you encounter or feature requests you have via [GitHub Issues](https://github.com/hezaerd/noyau/issues).
 
-## Prérequis
+### Installation
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)